ui-test-util/src/main/java/com/example/uitestutil/EspressoUtils.kt

```diff
@@ -1,6 +1,5 @@
 package com.example.uitestutil
 
-import android.util.Log
```

Comment:
you can revert this file we don’t need to rewrite existing functions

Reply:
@carmelo-iriti I believe that we need a function that asserts the existence of a button with a certain text, in order to verify opening of a proper PM, but if you think the test without that assertion is sufficient for us, then ok

Reply:
The question is: Is it possible to use a button that doesn’t exist? The check is implicit